wiki/User_title/id.md

```diff
@@ -0,0 +1,32 @@
+---
+tags:
+  - gelar
+  - gelar forum
+  - peringkat
+  - warna pengguna
+---
+
+# Gelar pengguna
+
+![Cuplikan laman profil Tom94](img/dev.png "Gelar \“osu!dev\” sebagaimana yang tersemat pada laman profil Tom94.")
+
+**Gelar pengguna** merupakan label atau tajuk khusus yang tersemat pada laman profil pemain. Suatu gelar pengguna pada umumnya disematkan oleh [osu!team](/wiki/People/The_Team) untuk menandakan status keanggotaan seseorang di dalam [gugus pengguna (*user group*)](/wiki/People/The_Team#user-groups) tertentu. Meskipun demikian, terdapat beberapa gelar yang [diberikan secara khusus](/wiki/People/Users_with_unique_titles) kepada pemain-pemain tertentu sebagai bentuk apresiasi atas pencapaian yang telah mereka raih dalam bidangnya masing-masing.
```

Comment:
```suggestion
**Gelar pengguna** merupakan label atau tajuk khusus yang tersemat pada laman profil pengguna. Suatu gelar pengguna pada umumnya disematkan oleh [osu!team](/wiki/People/The_Team) untuk menandakan status keanggotaan seseorang di dalam [gugus pengguna (*user group*)](/wiki/id/People/The_Team#user-groups) tertentu. Meskipun demikian, terdapat beberapa gelar yang [diberikan secara khusus](/wiki/id/People/Users_with_unique_titles) kepada pengguna-pengguna tertentu sebagai bentuk apresiasi atas pencapaian yang telah mereka raih dalam bidangnya masing-masing.
```

Reply:
`Pengguna` lebih efisien digunakan daripada `Pemain`, banyak kontributor yang hanya melakukan registrasi dan tidak pernah memainkan game sama sekali.

Reply:
o you don't really need to add the locale (i.e. `wiki/id/...` or `wiki/fr/...`) in the translated article orz, the osu! website has been programmed to auto-recognize what language you're currently browsing the site/the wiki in :x